css/index.css

```diff
@@ -1,3 +1,426 @@
+	display: inline-block;
+	width: 450px;
+	text-align: center;
+	padding: 0px 0px 40px;
+}
+
+ .intro h1 {
+	font-size: 50pt;
+	/* border: 1px dashed red; */
+}  
+
+
+.introimg {
+	width: 30%;
+	/* border: 1px dashed red; */
+	display: inline-block;
+	/* TRBL*/
+	padding: 20px 10px 10px 20px;
+}
+
+.intro button {
+	padding: 5px 40px 5px;
+	border: 1px solid black;
+	font-family: 'Titillium Web';
+}
+
+header p {
+	text-align: center;
+	border: 1px solid black;
+	width: 150px;
+}
+.codeimg {
+	width: 30%;
+	height: 30%;
+	/*TRBL*/
+	padding: 0 20px 20px 100px;
+}
+
+.top-content {
+	border-top: 1px solid black;
+	padding: 30px 0 30px;
+}
+
+
+.top-content .features {
+	display: inline-block;
+	/* border: 1px solid red; */
+	width: 49%;
+	margin-right: 5px;
+}
+
+.top-content .about {
+	display: inline-block;
+	/* border: 1px solid red; */
+	width: 49%;
+}
+
+.middle-img {
+	display: block;
+	width: 100%;
+}
+
+.bottom-content {
+	border-bottom: 1px solid black;
+	padding: 20px 0 30px;
+}
+
+.bottom-content .services {
+	display: inline-block;
+	width: 32%;
+	/* border: 1px solid red; */
+
+}
+
+.bottom-content .product {
+	display: inline-block;
+	width: 32%;
+	margin: 0px 10px 0px;
+}
+
+.bottom-content .vision {
+	display: inline-block;
+	width: 32%;
+}
+
+.contact-content {
+	margin-top: 30px;
+}
+footer p {
+	text-align: center;
+}
+
+/*****************QUERIES*******************/
+
+@media screen and (max-width: 800px) {
+    header{
+      
+        display: flex;
+        justify-content:space-around;
+        height: 100%;
+        align-items:center;
+        flex-direction: column;
+        /* width:100%; */
+        
+
+      }  
+
+    .logo{
+        order: -1;
+        margin-bottom:3%;
+        padding-top:5%;
+        
+
+    }
+   nav a{
+       display:flex;
+       align-content: stretch;
+
+       /* justify-content: space-around; */
+        /* margin-right:0%; 
+        margin-left:9%;  */
+
+       
+
+   }
+    nav{
+        margin-top:2%;
+        margin-bottom:5%;
+        display:flex;
+        justify-content: space-around;
+        display:flex;
+       align-content: stretch;
+        /* margin-left:7%; */
+
+        
+        
+    }
+
+    .introimg{
+        position:absolute;
+        visibility:hidden;
+    } 
+    .container{
+        width:95%;
+        padding-left:12%;
+       
+
+    }  
+    .middle-img{
+        /* padding-right:3%; */
+    }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+}
+
+@media screen and (max-width: 500px) {
+       .container{
+        width:100%;
+        /* padding-left:7%; */
+       
+
+    } 
+    header{
+      
+      display: flex;
+      width:100%;
+
+      height: 100%;
+      
+      flex-direction: column;
+      margin-left:10%;
+      
+    }  
+    .logo{
+        order: -1;
+        padding-bottom:6%;
+        
+
+    }
+    nav {
+        display:flex;
+        flex-direction: column;
+        
+        /* margin-top:2%; */
+        width:150%;
```

Comment:
Rather than increasing the width past 100% try to figure out what's constraining it from going across the full length at 100%;